### PR TITLE
I have a 25 kill streak (test merge recommended)

### PR DESCRIPTION
**Pull request by Dissentbay**
## About The Pull Request
<details>
<summary>
	you know what that means **TACTICAL NUKE INCOMING**
</summary>
<hr>
reminder that these changes are subject to change with feedback (insulting will make me nerf stuff harder), also this PR was not tested locally, so I'm sorry for any missed spots on the deletions 😔 

	
<hr>
</details>

## Changelog
:cl:
fix: nearly all church guns had their energy cost per shot raised (martyr is safe, dominion and it's cousin got an extra zero at the end, others got around a 25% cost per shot increase).
tweak: prospectors don't start with the thompson or homemade ak in their locker .They now can get a mare's leg and the 9mm luty instead. The odds of a strelki and a vector are reduced.
del: Baroque is out, hunters now can only use their recurve bow to one shot stuff.
del: stun church prayer (the one with 30min cooldown).

/:cl:

### CODE REVIEW

**Comment by Dissentbay** [comment]
of course I forgot something

File: code/modules/projectiles/guns/energy/laser/antebellum.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	suitable_cell = /obj/item/cell/medium/neotheology
 	projectile_type = /obj/item/projectile/plasma/light
 	fire_delay = 10
-	charge_cost = 100 //10 shots with basic 15 with level 2
+	charge_cost = 125 //10 shots with basic 15 with level 2
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/energy/laser/antebellum.dm:17]: please update the commit

File: code/modules/projectiles/guns/energy/plasma/excubitor.dm [hunk 2]
```diff
@@ -15,7 +15,7 @@
 	force = WEAPON_FORCE_BRUTAL
 	armor_penetration = ARMOR_PEN_MASSIVE
 	fire_delay = 10
-	charge_cost = 100 //10 shots with basic 15 with level 2
+	charge_cost = 125 //10 shots with basic 15 with level 2
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/energy/plasma/excubitor.dm:18]: please update the commit

File: code/modules/projectiles/guns/energy/laser/antebellum.dm (hunk 1 above)
Comment (Dissentbay) [reply code/modules/projectiles/guns/energy/laser/antebellum.dm:17]: my bad! updating it! and the others too, I think there is more of those 🤔

**Comment by Trilbyspaceclone** [comment]
im completely against the removal of the scavs strelki and vector from spawns.
I also really dislike the punchhole being added to their spawns

**Comment by Dissentbay** [comment]
I did it on Rebel's recommendation (the removal)
instead of leaving nothing instead I added other guns

**Comment by Trilbyspaceclone** [comment]
code/modules/trade/datums/trade_stations_presets/2-uncommon/KriosanConfederacy.dm:77:error: /obj/item/gun/projectile/boltgun/baroque: undefined type path


Also I understand your doing it on a recomination, but again im stating that **i** dislike the punchole being adding

And that the skrel and vector should stay as rare spawns

**Comment by Dissentbay** [comment]
hm, can I make it harder to spawn then?

**Comment by Dissentbay** [comment]
and the kriosan trader error is already looked at

File: code/modules/projectiles/guns/energy/laser/antebellum.dm (hunk 1 above)
Comment (Dissentbay) [reply code/modules/projectiles/guns/energy/laser/antebellum.dm]: fixed

File: code/game/objects/structures/crates_lockers/closets/secure/cargo.dm [hunk 3]
```diff
@@ -370,15 +372,16 @@
 				new /obj/item/gun/projectile/shotgun/pump(src)
 				new /obj/item/ammo_magazine/ammobox/shotgun/scrap_pellet(src)
 				new /obj/item/ammo_magazine/speed_loader_shotgun/empty(src)
-		if("GOOD_SMG")
-			if(prob(50))
+		if("SHORT_WILDCARDS")
+			if(prob(40))
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/secure/cargo.dm:376]:
```suggestion
			if(prob(20))
```

File: code/game/objects/structures/crates_lockers/closets/secure/cargo.dm [hunk 4]
```diff
@@ -184,15 +184,16 @@
 				new /obj/item/gun/projectile/shotgun/pump(src)
 				new /obj/item/ammo_magazine/ammobox/shotgun/scrap_pellet(src)
 				new /obj/item/ammo_magazine/speed_loader_shotgun/empty(src)
-		if("GOOD_SMG")
+		if("SHORT_WILDCARDS")
 			if(prob(50))
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/secure/cargo.dm:188]:
```suggestion
			if(prob(20))
```

**Comment by Dissentbay** [comment]
oh forgot to change the changelog

**Comment by Dissentbay** [comment]
should be all good to go 👍

**Comment by DorfGuy1** [comment]
maybe don't remove the baroque with no explanation given.

**Comment by Dissentbay** [comment]
> maybe don't remove the baroque with no explanation given.

straight nerfing it will just make it a worse armstrong, since reload is a bitch in it
plus hunters are all about using animal parts, so why have a gun that will gib most of the stuff you shoot at?
also the recurve bow, when modded and using proper arrows, already fills the niche and keeps more in the theme of hunters, and is way less prone to cause gibbing

the gun is gone, but there is an alternative in it's place, so you're not left empty handed

**Comment by DorfGuy1** [comment]
> > maybe don't remove the baroque with no explanation given.
> 
> straight nerfing it will just make it a worse armstrong, since reload is a bitch in it plus hunters are all about using animal parts, so why have a gun that will gib most of the stuff you shoot at? also the recurve bow, when modded and using proper arrows, already fills the niche and keeps more in the theme of hunters, and is way less prone to cause gibbing
> 
> the gun is gone, but there is an alternative in it's place, so you're not left empty handed

I'm pretty sure that some of the recurve bow mods are guild only.

**Comment by Dissentbay** [comment]
> I'm pretty sure that some of the recurve bow mods are guild only.

and that's bad because?

**Comment by Catalaria** [comment]
>dominion and it's cousin got an extra zero at the end
I completely understand the fact that Dominion and Purger had a whole lot of ammo than needed- but the extreme increase simply placed an high-cost, good quality gun directly to the lower tiers in one go. With the gun now draining 200 power from the battery- The purge can now only shoot three times. This will not fix anything but cause the Church to have one less thing to sell - as the purger is now essentially low tier, too cost-inefficient to produce much less to be sold to other people - and move on to search another meta or scrap the entire church-theme and use other department weaponry that is obviously cheaper, easier to obtain and BETTER.
if it had 8-to-10 shots with the average church battery, it would actually be balanced. But THREE shots?
No. This is half-assed.

**Comment by khornate01** [comment]
The purger and dominion are now the WORST plasmaguns in the entire game, despite being the expensive deluxe plasmaguns that come straight from the "best" plasmagun manufacturers in our lore. This change makes them so incredibly bad that they may as well not even exist because nobody in their right minds would print them knowing their new stats. 

I may go so far as to say that ALL the church energy weapon cost changes should be completely reverted. Church weapons being... tolerable to use... isn't a balancing issue. Prior to this PR, the purger and dominion weren't even meta. They are quite expensive and fairly powerful but still not as good as soteria or guild's best weapons. They are merely good enough that it doesn't feel like you're throwing mats in a hole when you print them. 

As it stands, the church meta doesn't even involve any of the primary weapons. It's mostly just melee and sidearms. What this change has essentially done is made the existing melee/martyr meta to be the only real option. So basically, this PR doesn't touch the meta at all, but removes its only competition. Even before this PR, the purger's DPS was only a tad higher than the halberd, and can't beat a properly sharpened one. Who cares if it can fire 20 times without reloading if it's not as good as something that never needs to reload at all? 

Personally I would undo all these power consumption increases. Up until right before this PR, the church's balance was in a pretty good place. New med changes made bleeding less dangerous, which made blades less of an uncontested champion. It's really as close to balanced as we've ever been. But you can bet with these changes, you'll never see a churchie using a plama rifle or lasgun again.

**Comment by Dissentbay** [comment]
here is the thing, everyone knew the purger and the dominion power cost was bullshit. So why no church member fixed it? If you don't want things changed in a way you don't like, be the first to change the stuff, so a better compromise can be achieved
I will nerf more stuff in other factions, things that people who main said factions should be able to acknowledge that are bullshit and should have nerfed themselves, but decided to ride on the op wave and hope no one capable of coding would change it.
Now is the time the nerfs will come, so not only church, but all factions should play smart and nerf the obvious stuff themselves before I get to start working on it

**Comment by StarDustFoxFox** [comment]
I understand the attempt but upon closer inspection and what I heard from other churchies, those changes made nothing but force church to rely on martyr, currently the martyr deals the same damage as the purger, 10 shots before burning out, you can fit 5 martyr in a medium pouch which in turn fits on a pocket, lightfall is meant to be a cog+ due to its cost, silver, wood and plasteel, while the cog itself uses only plastic, glass and steel, the purger and dominion are meant to be the cassad+, as cassad is also a plasma rifle which equally sets creatures and people on fire, but following this logic, not only the purger and dominion became worse than cassad, now a new unbalance has been made on church where everyone will start carrying full pouches of martyr, this change to energy cost did nothing but reinforce a martyr meta, which now makes it so church weaponry are useless due to their stats, once again reinforcing martyr, and even if you were to nerf martyr to have only 1 or 3 shots, people would still fit them inside a large or medium pouch, it won't solve anything other than make church grind harder to produce more biomatter to fuel martyr production, I even asked you to switch lightfall energy cost for the carpediem, but you did so changing the carpediem for blunderbuss, which is yet another useless gun of the church which no one uses, those changes don't tap the main problem, but rather makes it worse by all aspects.

Church now doesn't need to produce cells for their guns, they need to mass produce martyr and melee, which is a terrible balance due to martyr being equally strong as purger, and now even stronger, this PR broke the already messed balance of church weaponry and simply threw all guns out of the window and by accident buffed martyr.

**Comment by StarDustFoxFox** [comment]
I would suggest taking feedback from church channel on discord than do it without even knowing what church main players think of it.

**Comment by Dissentbay** [comment]
>  the purger and dominion are meant to be the cassad+, as cassad is also a plasma rifle which equally sets creatures and people on fire, but following this logic, not only the purger and dominion became worse than cassad, now a new unbalance has been made on church where everyone will start carrying full pouches of martyr, this change to energy cost did nothing but reinforce a martyr meta,

okay, now I see your point, I will get a pr changing the values in the plasma ones today to be more inline with the cassad, or maybe upping the cassad/psi counterpart

> I would suggest taking feedback from church channel on discord than do it without even knowing what church main players think of it.

I would suggest church players nerfing themselves, but here we are

**Comment by Dissentbay** [comment]
also it seems I should have taken a look at the martyr too, maybe upped the cost too to be more in line with plasma guns in general

once I finish talking with more church people I will get a verdict about it

**Comment by StarDustFoxFox** [comment]
Even if this buffs dominion and purger again to be equal to cassad in regard to charge cost, a lot in this PR doesn't even resembles balance, those changes were made by someone who doesn't play church and doesn't know what the church players think, most of the church players are either unable or not aware how to do PRs, and it cannot be simply "church mains should've nerfed it themselves", because you will see that most church players agreed that those changes needed to happen regarding their weaponry, most cases even asking coders to do such, but so far nothing, now this PR again won't change anything but reinforce a meta which is more cumbersome and boring than before, specially considering the mat cost of those weapons, guild reductor costs plasteel and silver, yet it is still a double tap gun, the mat cost wise is absurdly high even for its own sake, the same applies to church guns if not worse, as the purger and dominion costs plasteel, wood, silver and uranium, remember again that uranium is the hardest mat for church to find by themselves without having to trade or mine on space, guild reductor can be achieved by scraping maint walls dor plasteel and silver from dismantling machines, and you are aware of that, instead of nerfing the reductor you instead went to nerf a faction which you didn't ask for feedback nor even reads what they say most of the time, if this is the logic you follow, then the reductor should be nerfed to be 1 shot per large cell akin to how a potato battery currently does it, this PR again doesn't tap the proper problem of the church, it only worsens it, there is no gameplay change, there is just more grind for things which doesn't even make the gane more fun for people with or outside church, again I will say that lightfall is meant to be cog+ and purger and dominion as cassad+, carpediem needed this nerf regarding its energy cost, blunderbuss is still worthless even with the change for its energy cost due to damage and AP, even considering it also takes silver, most church guns take silver and gold if not rarer mats, those changes again only nerf those guns to the ground until someone shows up to fix it or ask this PR to be reverted.

If you want to nerf the guns of the church, would be best to ask church mains that are more than aware of how OP those guns are how it would be best to nerf em, because again, church is a department roleplay focused rather than gun nuts.

**Comment by Shirumic** [comment]
keep in mind the martyr is meant to be a high-powered, but disposable, emergency weapon. it's there as a tertiary weapon in case you're completely fucked and lost all your other weapons or can't reload them fast enough.

people were never supposed to be carrying twenty of them in their bag like some kind of pirate with flintlocks, but nerfs to the church without taking into account:
 how their arsenal works, 
 how people tend to use them, 
 and how changing one part of a church weapon might effect all of it.

so players change in strange ways because the game does not accommodate playing with reasonable intuition and sanity, but mass producing one or two meta weapons because everything else they have can't solve any of their problems.

**Comment by Dissentbay** [comment]
first, don't compare reductor with the plasmas, you're shooting yourself on the foot if church should be the a good source of guns in the colony, gameplay-wise church is meant to be "independent" from the colony, not superior in any way, otherwise you kill the purpose of the church even being in the colony and not another lodge, plus Guild has the "best engineers in sol" lore stuff behind them to back it up in case of crafting stuff

second, reductor is on the chopping block too, but the issue of the reductor is that a lot of other guild guns use the reductor as base, so you don't just nerf the reductor, you're also nerf the myrm (that is already in a good spot) and the batko(only source of and AMR that is not in the armory or requires prospies) in a multiplicative way, changing the power cost of the dominion and the purger didn't also change the martyr, did it? so blame past code for some nerfs coming faster than others, since there is more to come to all factions (i think service is the only who won't get a nerf, since they already had it bad enough)

but yeah, give me some time to sort this out, I happen to do other stuff than code for the server

**Comment by StarDustFoxFox** [comment]
Church has the lore of being excellent producers of plasma weaponry, so much that the hydrogen guns if soteria state in their desc that they are an attempt of Mkne to copy church designs, this sounds enough to say church knows how to make plasma weaponry, if not better than soteria even

And even so if depending on lore, it is a mess by itself as no one knows what church is even meant to be on sojourn, lore doesn't hold anything as when guild is considered best engineers of sol buy they can't make normal cells for their own guns.

Again I will state that this needs to be reviewed with church and other lore devs even as this is a mess.

**Comment by Dissentbay** [comment]
> Again I will state that this needs to be reviewed with church and other lore devs even as this is a mess.

Again, this is already happening

**Pull request merged**